packages/components/src/card/index.jsx

```diff
@@ -47,13 +35,25 @@ class Card extends PureComponent {
 			...props
 		} = this.props;
 
+		const elementClass = classNames(
+			'card',
+			className,
+			{
+				'is-card-link': displayAsLink || href,
+				'is-clickable': this.props.onClick,
```

Comment:
Could destructure alongside `className` and friends.

Reply:
That would remove it from the props that are spread below, requiring it to be passed back (potentially as undefined) to the rendered JSX. It seems preferable to leave it here.